### PR TITLE
Migration to rocketbase. Change ssd rocket.

### PR DESCRIPTION
**Pull request by IgorSusmelj**
Switch from rockethub package to rocketbase pip package.
Replace ssd Rocket experiments with Retinanet101 Rocket.

Signed-off-by: Igor Susmelj isusmelj@gmail.com

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,21 +10,24 @@ We are calling those new way of packaging Deep Learning models: __Rockets__.
 __Welcome to the Rockets Scientists Community!!!__
 
 ## Install the repositories
-We recommend you to use an isolated Python environement such as [virtualenv](https://virtualenv.pypa.io/en/latest/) or [conda](https://docs.conda.io/en/latest/) with at least __Python 3.5__. Then you can use the following lines of code:
+We recommend you to use an isolated Python environement such as [virtualenv](https://virtualenv.pypa.io/en/latest/) or [conda](https://docs.conda.io/en/latest/) with at least __Python 3.6__. Then you can use the following lines of code:
```
Comment (LucasVandroux) [inline README.md:13]: Why did you change it to python 3.6? The default python version on most system is still 3.5. And it should still work with 3.5.

File: README.md [hunk 2]
```diff
@@ -10,21 +10,24 @@ We are calling those new way of packaging Deep Learning models: __Rockets__.
...
 ```
-git clone https://github.com/LucasVandroux/PyTorch-Rocket-YOLOv3-RetinaNet-SSD
-cd PyTorch-Rocket-YOLOv3-RetinaNet-SSD
-pip install -r requirements.txt
+git clone https://github.com/LucasVandroux/PyTorch-Rocket-YOLOv3-RetinaNet50-RetinaNet101
+cd PyTorch-Rocket-YOLOv3-RetinaNet50-RetinaNet101
+pip install rocketbase
```
Comment (LucasVandroux) [inline README.md:17]: As rocketbase, contains all of the needed dependencies we don't need the requirements.txt anymore.

**Review by LucasVandroux — COMMENTED** [review]
Left two comments. Please have a look.

File: README.md (hunk 1 above)
Comment (IgorSusmelj) [reply README.md:13]: officially, the PIP package only supports python 3.6+

However, on some systems, the code might also work on lower python versions.

**Review by LucasVandroux — APPROVED** [review]
Good job Igor!
